Archive/Imo/Imo1998Q2.lean

```diff
@@ -130,19 +130,17 @@ theorem A_fibre_over_contestant_card (c : C) :
       ((A r).filter fun a : AgreedTriple C J => a.contestant = c).card := by
   rw [A_fibre_over_contestant r]
   apply Finset.card_image_of_injOn
-  -- Porting note (#10936): used to be `tidy`. TODO: remove `ext` after `extCore` to `aesop`.
-  unfold Set.InjOn; intros; ext; all_goals aesop
+  -- Porting note (#10936): used to be `tidy`.
```

Comment:
I guess this porting note is no longer actionable (the `tidy` tactic was not ported; aesop is meant to replace it), so tend towards removing it. I'll let a maintainer comment.

Reply:
Yes, please remove it.

Reply:
done